Add interactive params embed option (#795)
vimeo/player.js: README.md

autopause         | `true`   | Pause this video automatically when another one plays.
autoplay          | `false`  | Automatically start playback of the video. Note that this won’t work on some devices.
background        | `false`  | Enable the player's background mode which hides the controls, autoplays and loops the video (available to  Plus, PRO, or Business members).
byline            | `true`   | Show the byline on the video.
color             | `00adef` | Specify the color of the video controls. Colors may be overridden by the embed settings of the video.
controls          | `true`   | This parameter will hide all elements in the player (play bar, sharing buttons, etc) for a chromeless experience. ⚠️Warning: When using this parameter, the play bar and UI will be hidden. To start playback for your viewers, you'll need to either enable autoplay or use our player SDK to start and control playback. (available to Plus, PRO, or Business members)
dnt               | `false`  | Block the player from tracking any session data, including cookies.
height            |          | The exact height of the video. Defaults to the height of the largest available version of the video.

> interactiveparams |          | For interactive videos only that utilize live elements, this will contain key/value pairs for dynamic values to be passed into the live element. Ex: `[key1=value1,key2=value2]` 

Reviewer: ```suggestion
interactiveparams |          | Key-value pairs representing dynamic parameters that are utilized on interactive videos with live elements. Ex: `[key1=value1,key2=value2]`
```

We should also ask Player Tools / Alex C whether there's an article or documentation that we can link to from here that can provide more context.

Author: Per Conrad, they don't have front-facing documentation for this feature yet, and he recommended we don't link to the older Wirewax docs, so we may have to circle back at a later date to add a doc link.
